commands/azureCommands/delete-image.ts

```diff
@@ -0,0 +1,57 @@
+import { Registry } from "azure-arm-containerregistry/lib/models";
+import * as vscode from "vscode";
+import { dockerExplorerProvider } from '../../dockerExtension';
+import { UserCancelledError } from "../../explorer/deploy/wizard";
+import { AzureImageNode, AzureRepositoryNode } from '../../explorer/models/azureRegistryNodes';
+import { reporter } from '../../telemetry/telemetry';
+import * as acrTools from '../../utils/Azure/acrTools';
+import { AzureImage } from "../../utils/Azure/models/image";
+import { Repository } from "../../utils/Azure/models/repository";
+import { AzureUtilityManager } from '../../utils/azureUtilityManager';
+import * as quickPicks from '../utils/quick-pick-azure';
+
+const teleCmdId: string = 'vscode-docker.delete-ACR-Image';
+
+/** Function to delete an Azure hosted image
+ * @param context : if called through right click on AzureImageNode, the node object will be passed in. See azureRegistryNodes.ts for more info
+ */
+export async function deleteAzureImage(context?: AzureImageNode): Promise<void> {
+    if (!AzureUtilityManager.getInstance().waitForLogin()) {
+        vscode.window.showErrorMessage('You are not logged into Azure');
+        throw new Error('User is not logged into azure');
+    }
+    let registry: Registry;
+    let repoName: string;
+    let tag: string;
+
+    if (!context) {
+        registry = await quickPicks.quickPickACRRegistry();
+        const repository: Repository = await quickPicks.quickPickACRRepository(registry, 'Choose the Repository of the image you want to delete');
+        repoName = repository.name;
+        const image: AzureImage = await quickPicks.quickPickACRImage(repository, 'Choose the Image you want to delete');
+        tag = image.tag;
+    } else {
+        registry = context.registry;
+        let wholeName: string[] = context.label.split(':');
+        repoName = wholeName[0];
+        tag = wholeName[1];
+    }
+
+    const shouldDelete = await quickPicks.confirmUserIntent(`Are you sure you want to delete ${repoName}:${tag}? Enter Yes to continue: `);
```

Comment:
Hmm not sure. The idea behind the type yes is that since delete is an undoable function we want to make sure that users are completely certain, it seemed that towards that regards a typed confirmation wad best but we can try the other approach, thanks for the feedback! Ill try to start on it by tomorrow

Reply:
Yeah, I get it.  The portal does this, too.  We haven't in any of our extensions.  That's why I asked for Matt's opinion.  Here's an example: https://github.com/Microsoft/vscode-cosmosdb/blob/32a5e091475496c3b4b9d9ba8ae2d2999564f242/src/commands/deleteCosmosDBAccount.ts#L15

Reply:
@estebanreyl Could you change "delete image" so that it uses the simple yes/cancel prompt like I pointed to?  thx.  Delete repository and registry should continue using your current method.

Reply:
perfect, Ill update accordingly